nebula_ros/src/hesai/hesai_ros_wrapper.cpp

```diff
@@ -103,30 +103,30 @@ nebula::Status HesaiRosWrapper::DeclareAndGetSensorConfigParams()
       default_value = 600;
     }
     config.rotation_speed =
-      declare_parameter<uint16_t>("rotation_speed", default_value, descriptor);
```

Comment:
Please remove `deault_value` in lines `94`, `99` and `103` as well

Reply:
https://github.com/tier4/nebula/pull/155/commits/16a2a493d350c00c877d60e7eb4e4cfbde1cd13b